docs/GHA.md

```diff
@@ -1,60 +1,66 @@
-
-The `permissions` key allows for the interaction with commit `statuses`. For Buildkite to publish commit statuses for builds based on commits and pull requests on pipeline builds: the [GitHub App](https://buildkite.com/docs/integrations/github#connect-your-buildkite-account-to-github-using-the-github-app) must be added to the respective GitHub organization for statuses to appear based on a build's outcome. The GitHub App can be configured with access to all repositories within a GitHub organization - or a select number. 
-
-### run_name
-
-This key sets the name of a GitHub Action workflow's run. A `run_name` is specific to this integration: within Buildkite, when pipeline builds are created - a build message can be specified (contained in each job's environment as `BUILDKITE_MESSAGE`). The value is empty when a message is not set - and on events from source control, is the commit's title. A build message can also be set when creating a build through the [REST](https://buildkite.com/docs/apis/rest-api/builds#create-a-build) and [GraphQL](https://buildkite.com/docs/apis/graphql/schemas/mutation/buildcreate) APIs.
\ No newline at end of file
+> [!NOTE]  
+> When Buildkite builds are ran; each created command step inside the pipeline are ran as a [job](https://buildkite.com/docs/pipelines/defining-steps#job-states) that will be distributed and assigned to the matching agents meeting its specific queue and/or tag [targeting](h.ttps://buildkite.com/docs/pipelines/defining-steps#targeting-specific-agents). Each job is run within its own seperate environment, with potentially different environment variables (for example those defined at [step](https://buildkite.com/docs/pipelines/command-step#command-step-attributes) level) - and is not always gauarnteed to run on the same agent depending on targeting rules specified/agent fleet setup.
+
+| Key | Supported? | Notes |
+| --- | ---------- | ----- |
+| `jobs.<id>.concurrency` | 🟡 | The `group` name inside a `concurrency` definition inside a job maps to the `concurrency_group` [key](https://buildkite.com/docs/pipelines/controlling-concurrency#concurrency-groups) available within Buildkite. The `cancel-in-progress` optional value maps to the Buildkite pipeline setting of [Cancel Intermediate Builds](https://buildkite.com/docs/pipelines/skipping#cancel-running-intermediate-builds)  Buildkite also allows a upper limit on how much jobs are created through a single step  definition with the `concurrency` key: which is set as `1` by default (there isn't a translatable key within a GitHub Action workflow). |
```

Comment:
There is no period or separation of the last section, but I believe that there should be 3 distinct paragraphs altogether as they show something translated, something that requires manual intervention and something new that people may be interested in.

```suggestion
| `jobs.<id>.concurrency` | 🟡 | The `group` name inside a `concurrency` definition inside a job maps to the `concurrency_group` [key](https://buildkite.com/docs/pipelines/controlling-concurrency#concurrency-groups) available within Buildkite.

The `cancel-in-progress` optional value maps to the Buildkite pipeline setting of [Cancel Intermediate Builds](https://buildkite.com/docs/pipelines/skipping#cancel-running-intermediate-builds) that you will need to activate in the pipeline settings.
Buildkite also allows a upper limit on how much jobs are created through a single step  definition with the `concurrency` key: which is set as `1` by default (there isn't a translatable key within a GitHub Action workflow). |
```

Reply:
Had to do this via the line break `<br/>` way but added here, and to the topmost note